lib/tasks/schools/match_establishments.rake

```diff
@@ -0,0 +1,16 @@
+namespace :school do
+  desc 'Update establishment_id column for all active Schools in England and Wales'
+  task :match_establishments, [:path] => :environment do |_t, _args|
+    stats = { perfect: 0, la_plus_en: 0, unmatched: 0 }
+    update = {}
+
+    School.active.where(country: [0, 2]).find_each do |sch| # Only do active schools in England and Wales
```

Comment:
You can use :england and :wales here rather than the numbers, which will be clearer.